pkg/configure/configure.go

```diff
@@ -74,6 +74,13 @@ func ReconfigureHandler(k8sClient client.Client) func(w http.ResponseWriter, r *
 		// Infer managed/unmanaged components from the given `config.yaml`.
 		newComponents := []v1.Component{}
 		for _, component := range quay.Spec.Components {
+
+			// HPA and Monitoring don't have fields associated with them so we skip. Route should not change based on config either since fields are optional when managed.
+			if component.Kind == v1.ComponentHPA || component.Kind == v1.ComponentMonitoring || component.Kind == v1.ComponentRoute {
```

Comment:
This is only in the reconfigure request coming from the config editor. You can still un/manage the component in the spec, but since these fields don't have config values associated with them, there is no way to infer whether they are managed or unmanaged based on the presence of a field in the config.